bigbluebutton-html5/imports/ui/components/actionsbar/Actionsbar.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './styles.scss';
+
+export default class Actionsbar extends Component {
```

Comment:
I think so. The only "bar" that isn't two words is "sidebar", but that is just because it's a real word. https://en.wikipedia.org/wiki/Sidebar_(computing) vs. https://en.wikipedia.org/wiki/Navigation_bar